From 849bc730b7b552b2c84b538f57f487bcf9d18ed1 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:36 -0400
Subject: [ethernet] igb: remove unnecessary break after goto

Message-id: <1409062359-13063-79-git-send-email-sassmann@redhat.com>
Patchwork-id: 89193
O-Subject: [RHEL7.1 PATCH 78/81] igb: remove unnecessary break after goto
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit a4a07624927743df7f4414e7f368b49ff19271b9)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/e1000_82575.c b/drivers/net/ethernet/intel/igb/e1000_82575.c
index ee74f95..72b454c 100644
--- a/drivers/net/ethernet/intel/igb/e1000_82575.c
+++ b/drivers/net/ethernet/intel/igb/e1000_82575.c
@@ -837,7 +837,6 @@ static s32 igb_get_phy_id_82575(struct e1000_hw *hw)
   default:
    ret_val = -E1000_ERR_PHY;
    goto out;
-   break;
   }
   ret_val = igb_get_phy_id(hw);
   goto out;
-- 
1.7.1